pupil-spa/src/app/check/check.component.ts

```diff
@@ -48,11 +49,15 @@ export class CheckComponent implements OnInit {
 
   manualSubmitHandler(answer: string) {
     // console.log(`check.component: manualSubmitHandler(): ${answer}`);
+    const answerSet = { factor1: this.question.factor1, factor2: this.question.factor2, answer: parseInt(answer, 10) };
```

Comment:
I think should be a string here as we need to record leading zero's `in the answer`.